src/Files.App/Helpers/Navigation/NavigationHelpers.cs

```diff
@@ -407,8 +422,11 @@ private static async Task<FilesystemResult> OpenFile(string path, IShellPage ass
 								// Now launch file with options.
 								var storageItem = (StorageFile)await FilesystemTasks.Wrap(() => childFile.Item.ToStorageFileAsync().AsTask());
 
-								if (storageItem is not null)
-									launchSuccess = await Launcher.LaunchFileAsync(storageItem, options);
+								if (!FileExtensionHelpers.IsPythonFile(storageItem.Path))
+								{
+									if (storageItem is not null)
```

Comment:
I think the null check should be before checking the file extension

Reply:
My apologies, this python check needs to be removed as the code path will no longer bring it here.
The code path for the python files should end in the function "OpenItemsWithPythonAsync", it will not make it here.